apps/financial-aid/web-veita/src/components/MunicipalityAdminSettings/MunicipalityAdminSettings.tsx

```diff
@@ -250,6 +266,29 @@ const MunicipalityAdminSettings = ({ currentMunicipality }: Props) => {
         />
       ),
     },
+    {
+      headline: 'Desemberstyrkur',
+      smallText: 'Upphæð er bætt við þegar umsókn er samþykkt í nóvember',
+      component: (
+        <NumberInput
+          id={`input-desember`}
+          name={`decemberCompensation`}
+          label="Desemberstyrkur"
+          maximumInputLength={6}
+          value={state.decemberCompensation.toString()}
```

Comment:
Should this be postfixed with ' kr. for clarity?'

Reply:
We just finished a meeting and now it will be procentage